### PR TITLE
Fix grammar and add some initials

### PR DESCRIPTION
**Pull request by claucece**
After reviewing for a second time the document, I decided to do some grammar fixes as well. Hope it is helpful.
cc./ @beurdouche

### CODE REVIEW

File: draft-ietf-mls-protocol.md [hunk 1]
```diff
@@ -344,23 +344,23 @@ the SP provides the following services:
...
-members, and to remove existing members. These messages produce new shared
-secrets which are causally linked to their predecessors, forming a logical
-Directed Acyclic Graph (DAG) of states.
+secret. Members exchange _Commit_ messages for post-compromise security, to
+add new members, and to remove existing members. These messages produce
+new shared secrets which are causally linked to their predecessors, forming
+a logical Directed Acyclic Graph (DAG) of states.
 
-The protocol algorithms we specify here follow. Each algorithm specifies
+The protocol algorithms we specify here as follow. Each algorithm specifies
```
Comment (Bren2010) [inline draft-ietf-mls-protocol.md:363]: -as

File: draft-ietf-mls-protocol.md [hunk 2]
```diff
@@ -344,23 +344,23 @@ the SP provides the following services:
...
 sends this directly to all the new members, who can use the information
 it contains to set up their own group state and derive a shared
-secret. Members exchange Commit messages for post-compromise security, to add new
-members, and to remove existing members. These messages produce new shared
-secrets which are causally linked to their predecessors, forming a logical
-Directed Acyclic Graph (DAG) of states.
+secret. Members exchange _Commit_ messages for post-compromise security, to
+add new members, and to remove existing members. These messages produce
+new shared secrets which are causally linked to their predecessors, forming
+a logical Directed Acyclic Graph (DAG) of states.
```
Comment (Bren2010) [inline draft-ietf-mls-protocol.md:361]: Wrapping text pollutes "git blame" unnecessarily. We don't have any real line length

File: draft-ietf-mls-protocol.md [hunk 3]
```diff
@@ -97,13 +97,13 @@ Instructions are on that page as well. Editorial changes can be
...
 a way to derive shared symmetric encryption keys. For two parties,
 this problem has been studied thoroughly, with the Double Ratchet
-emerging as a common solution {{doubleratchet}} {{signal}}.
-Channels implementing the Double Ratchet enjoy fine-grained forward secrecy
-as well as post-compromise security, but are nonetheless efficient
-enough for heavy use over low-bandwidth networks.
+Algorithm emerging as a common solution {{doubleratchet}} {{signal}}.
+Channels implementing the Double Ratchet Algorithm enjoy fine-grained
+forward secrecy, as well as post-compromise security; but are,
+nonetheless, efficient enough for heavy use over low-bandwidth networks.
```
Comment (Bren2010) [inline draft-ietf-mls-protocol.md:106]: I think the punctuation was fine before

File: draft-ietf-mls-protocol.md [hunk 4]
```diff
@@ -1325,8 +1322,8 @@ SHOULD match at least the security level of the ciphersuite in use in
 the group.
 
 Note that, as a PSK may have a different lifetime than an update, it
-does not necessarily provide the same FS or PCS guarantees than
-a Commit message.
+does not necessarily provide the same Forward Secrecy (FC) or Post-Compromise
```
Comment (Bren2010) [inline draft-ietf-mls-protocol.md:1325]: FC -> FS

File: draft-ietf-mls-protocol.md [hunk 5]
```diff
@@ -1678,19 +1675,19 @@ ciphersuite in use.
 
 # Group Creation
 
-A group is always created with a single member, the "creator".  The other
-members are added when the creator effectively sends itself an Add proposal and
-commits it, then sends the corresponding Welcome message to the new
-participants.  These processes are described in detail in {{add}}, {{commit}},
+A group is always first created with a single member, the "creator".  Other
+members are added when the creator effectively sends itself an _Add_ proposal
+and commits it, and then sends the corresponding _Welcome_ message to the
```
Comment (Bren2010) [inline draft-ietf-mls-protocol.md:1680]: Two options:
- sends itself an Add and commits, then (how it was)
- sends itself an Add, commits, and then

File: draft-ietf-mls-protocol.md [hunk 6]
```diff
@@ -1703,35 +1700,35 @@ The creator of a group MUST take the following steps to initialize the group:
...
 
-The recipient of a Welcome message processes it as described in
+The recipient of a _Welcome_ message processes it as described in
 {{welcoming-new-members}}.
 
 In principle, the above process could be streamlined by having the
 creator directly create a tree and choose a random value for first
 epoch's epoch secret.  We follow the steps above because it removes
-unnecessary choices, by which, for example, bad randomness could be
+unnecessary choices, when, for example, bad randomness could be
```
Comment (Bren2010) [inline draft-ietf-mls-protocol.md:1720]: "by which" or "where"

File: draft-ietf-mls-protocol.md [hunk 7]
```diff
@@ -2692,8 +2690,8 @@ updated root key after an Commit operation, it cannot compute any
 derived secrets.
 
 In the case where the client could have been compromised (device
-loss...), the client SHOULD signal the delivery service to expire
-all the previous KeyPackages and publish fresh ones for PCS.
+loss, for example), the client SHOULD signal to the delivery service
```
Comment (Bren2010) [inline draft-ietf-mls-protocol.md:2693]: -to

File: draft-ietf-mls-protocol.md [hunk 8]
```diff
@@ -97,13 +97,13 @@ Instructions are on that page as well. Editorial changes can be
 managed in GitHub, but any substantive change should be discussed on
 the MLS mailing list.
 
-A group of users who want to send each other encrypted messages needs
+A group of users who want to send each other encrypted messages need
```
[the remark this answers is not in the document]
Comment (claucece) [reply draft-ietf-mls-protocol.md:100]: Oh, yes! Thanks! I got confused by this: https://english.stackexchange.com/questions/39838/is-group-singular-or-plural

**Comment by claucece** [comment]
Mmm.. as the PR has too many line fixes, I'm going to open a new one with only the grammar changes.

**Pull request closed without merging**